index.d.ts

```diff
@@ -0,0 +1,61 @@
+import { Readable } from 'readable-stream'
+
+// TODO: can typescript resolve this, if not in our dependencies?
+import { AbstractLevel, AbstractIteratorOptions, AbstractKeyIteratorOptions, AbstractValueIteratorOptions } from 'abstract-level'
```

Comment:
As written

Reply:
Yeah, this doesn't work.

@MeirionHughes what's the right way to do this? I want to reference types from another npm package without adding it to dependencies here. If that's not possible, then what's the conventional TS way to deal with this?